engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkStreamingDeploymentManagerProvider.scala

```diff
@@ -41,4 +46,44 @@ object FlinkStreamingDeploymentManagerProvider {
     new FlinkStreamingDeploymentManagerProvider().createDeploymentManager(ModelData(typeConfig), typeConfig.deploymentConfig)
   }
 
+}
+
+object FlinkStreamingPropertiesConfig {
+
+  lazy val properties: Map[String, AdditionalPropertyConfig] =
+    Map(parallelismConfig, spillStateConfig, asyncInterpretationConfig, checkpointIntervalConfig)
+
+  private val parallelismConfig: (String, AdditionalPropertyConfig) = "parallelism" ->
+    AdditionalPropertyConfig(
+      defaultValue = None,
+      editor = Some(StringParameterEditor),
+      validators = Some(List(MandatoryParameterValidator, LiteralIntegerValidator, MinimalNumberValidator(1))),
+      label = Some("Parallelism"))
+
+  private val spillStateConfig: (String, AdditionalPropertyConfig) = "spillStateToDisk" ->
+    AdditionalPropertyConfig(
+      defaultValue = None,
+      editor = Some(BoolParameterEditor),
+      validators = None,
+      label = Some("Spill state to disk"))
+
+  private val asyncPossibleValues = List(
+    FixedExpressionValue("false", "Synchronous"),
+    FixedExpressionValue("true", "Asynchronous"),
+    FixedExpressionValue("", "Server default"))
```

Comment:
The first option on the list is considered the default. I think that we should have "" as first